src/components/layout/MessageIndicator.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+
+// Material UI Components
+import Grid from '@material-ui/core/Grid';
+import Typography, { TypographyProps } from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    padding: '15px 5px',
+    height: '100%',
+  },
+  header: {
+    color: theme.palette.colors.text.light,
+  },
+}));
+
+export type MessageIndicatorProps = {
+  header: string;
+  subheader: string;
+  color?: TypographyProps['color'];
+  variant?: TypographyProps['variant'];
+};
+
+const MessageIndicator = ({ header, subheader, color, variant }: MessageIndicatorProps) => {
+  const classes = useStyles();
+  return (
+    <Grid className={classes.root} container direction='column' justify='center' wrap='nowrap'>
+      <Typography align='center' className={classes.header} color={color || 'inherit'} gutterBottom variant={variant || 'h6'}>
```

Comment:
Default props bør legges der man henter ut propsene:
```js
const MessageIndicator = ({ header, subheader, color = 'inherit', variant = 'h3' }: MessageIndicatorProps) => {
```

```suggestion
      <Typography align='center' className={classes.header} color={color} gutterBottom variant={variant}>
```